src/test/java/com/rultor/agents/daemons/EndsDaemonITCase.java

```diff
@@ -103,6 +106,37 @@ public void readsExitCodeCorrectly() throws IOException {
     }
 
     /**
+     * EndsDaemon can deprecateDefaultImage.
+     * @throws IOException In case of error.
+     * @todo @todo #754:30min Implement a deprecation message at the start
+     *  of the process if the project is using the default image
+     *  'yegor256/rultor'.
```

Comment:
@sebing I see the risk that parallel development may cause some code duplication or two substantially different solutions for very similar problems. We are not in rush with this issue, that's why I propose one puzzle and implementer can decide whether it is possible to complete in one go or there will be another puzzle.